CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
+## \[v0.5.0\]
+
+### Added
+
+- Model Preparation Algorithm (MPA)
+  - Add tiling in rotated detection (<https://github.com/openvinotoolkit/training_extensions/pull/1420>)
+  - Add Cythonize AugMixAugment (<https://github.com/openvinotoolkit/training_extensions/pull/1478/>)
```

Comment:
Let's remove MPA. Maybe we can just move-left 2 "Add" items?

Reply:
Removed MPA category